colonel/wrapper/wrapper.py

```diff
@@ -5,12 +5,66 @@
 import tempfile
 from typing import Optional
 import logging
+import pandas as pd
 
 from colonel.wrapper.errors import SimulatorExecutableNotFound
 from colonel.wrapper.config import CDPP_BIN_PATH
 from colonel.models import Model
 
 
+# This object should contain the following properties:
+# - Whether or not the simulation was successful (Or maybe this should raise an error)
+# - The parsed logs
+# - The parsed output
+# - Elapsed simulation time
+# - Real time that the simulation took to be completed
+class SimulationResult:
+    TIME_COL = 'time'
+    PORT_COL = 'port'
+    VALUE_COL = 'value'
+    MESSAGE_TYPE_COL = 'message_type'
+    MODEL_ORIGIN_COL = 'model_origin'
+    MODEL_DEST_COL = 'model_dest'
+
+    def __init__(self, process_result, main_log_path=None, output_path=None):
+        self.process_result = process_result
+        self.main_log_path = main_log_path
+        self.output_path = output_path
+        self.output_df = SimulationResult.parse_output_file(output_path)
+        self.logs_dfs = SimulationResult.parse_main_log_file(main_log_path)
+
+    def successful(self):
+        return self.process_result.returncode == 0
+
+    @classmethod
+    def parse_output_file(cls, file_path):
+        return pd.read_csv(file_path, delimiter=r'\s+',
+                           names=[cls.TIME_COL, cls.PORT_COL, cls.VALUE_COL])
+
+    @classmethod
+    def parse_main_log_file(cls, file_path):
+        log_file_per_component = {}
+        parsed_logs = {}
+        with open(file_path, 'r') as main_log_file:
+            # Ignore first line
+            main_log_file.readline()
+            for line in main_log_file:
+                name, path = line.strip().split(' : ')
+                log_file_per_component[name] = path
+        for logname, filename in log_file_per_component.items():
+            parsed_logs[logname] = pd.read_csv(filename,
+                                               delimiter=r' /\s+',
+                                               engine='python',  # C engine doesnt work for regex
+                                               names=[0, 1,  # Not sure what first two cols are
```

Comment:
We should change or drop this columns if unused. If not, they will be accessible to the user, and confuse him/her.